client/a8c-for-agencies/sections/team/primary/team-invite/index.tsx

```diff
@@ -35,9 +36,20 @@ export default function TeamInvite() {
 
 		if ( ! username ) {
 			setError( translate( 'Please enter a valid email or WordPress.com username.' ) );
+			dispatch(
+				recordTracksEvent( 'calypso_a4a_team_invite_error', {
+					error: 'empty_username',
+				} )
+			);
```

Comment:
I'm curious if we need to track this. Not sure if this will come in handy in any case. What do you think?

Reply:
Actually, in some Calypso implementations, we track both success and errors. I suggest keeping it; it might be handy for funneling.

Reply:
I was curious, too, about this tracking level of details. And empty form field does not seem something to track. I don't see a potential funnel at this level.